sirepo/template/srw.py

```diff
@@ -2398,6 +2448,7 @@ def _set_parameters(v, data, plot_reports, run_dir, qcall=None):
     v.setupMagneticMeasurementFiles = (
         plot_reports or is_for_rsopt
     ) and _SIM_DATA.srw_uses_tabulated_zipfile(data)
+    v.isBeamlineAnimation = "beamlineAnimation" in report
```

Comment:
No need for this.

Reply:
[25d3a7f](https://github.com/radiasoft/sirepo/pull/6186/commits/25d3a7f080bc2c87175e2548e98ee2930cc39eaf)